src/pages/AdminManageDul.js

```diff
@@ -326,11 +335,14 @@ class AdminManageDul extends Component {
                     button({
                       id: election.consentId + "_btn_editDUL",
                       className: "cell-button hover-color",
-                      onClick: this.editDul(election)
+                      onClick: () => this.editDul(election)
```

Comment:
review this, every click invokes a new instance ...

Reply:
This is done this way due to, onClick listener expects ```this.editDul``` be a function, if we pass the  instance value to the function the app enters into an infinite loop.